src/modules/src/position_estimator_altitude.c

```diff
@@ -51,7 +51,11 @@ static struct selfState_s state = {
   .estimatedZ = 0.0f,
   .velocityZ = 0.0f,
   .estAlphaZrange = 0.90f,
-  .estAlphaAsl = 0.997f,
+  #ifdef IMPROVED_BARO_Z_HOLD
+    .estAlphaAsl = 0.90f,
+  #else
+    .estAlphaAsl = 0.997f,
```

Comment:
I do not see a reason for keeping the old value, it seems to be only used for baro-based estimation

Reply:
Yeah you are right, let's up it to the new value. I don't think it will matter that much for the rest of the estimation as for the zranger